src/sync/installation.ts

```diff
@@ -313,25 +313,56 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 		statsd.increment(metricTaskStatus.complete, [`type: ${nextTask.task}`]);
 
 	} catch (err) {
-		await handleBackfillError(err, logger, scheduleNextTask, data, task, repositoryId, subscription, nextTask);
+
```

Comment:
I've extrated some code from `handleBackfillError` into functions `ignoreCurrentRepo` and `failCurrentRepoAndContinue`. 

This is to make the code testable without changing much

Reply:
I'm not sure I understand the point of doing these changes.  Maybe it's making it easier to test, but it's also making it extremely hard to read and breaks multiple software practices.

Can we please keep the functionality as it was?

Reply:
The point is to make it testable. How would you suggest to test it otherwise?

Reply:
As we discussed. I was able to achieve the same result by using jest.spyOn on installation module functions